starknet/src/tests/mocks/account.cairo

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: MIT
```

Comment:
im not sure whether there is a way to add a contract in a dependency to the compilation artifacts. So just directly added the account  code here.

Reply:
figured out how to do it